openmc/settings.py

```diff
@@ -101,6 +101,13 @@ class Settings(object):
         Coordinates of the lower-left point of the Shannon entropy mesh
     entropy_upper_right : tuple or list
         Coordinates of the upper-right point of the Shannon entropy mesh
+    tabular_legendre : dict
```

Comment:
I kind of like it the way it is since they are directly linked together in this format here.